src/img/girlscodelincoln-nav.svg

```diff
@@ -0,0 +1,180 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Is there any reason why we're not using this SVG? If we don't want it, we can probably get rid of it

Reply:
It's only there as a source for the PNGs. It's not used on the site. I can remove it or move it out of the src directory.

Reply:
I'd say remove it unless we're using it. Otherwise, we could actually use the svg as the source. Maybe we could add in the README to note our google drive link to our assets?

Reply:
OK. I'll work on cleaning up the SVG by removing the Inkscape specific XML and removing the transform.

Reply:
I've used this in the past and it works well
https://www.svgminify.com/

Reply:
Thanks, that's slick. It did not remove the transform node but it cleaned up the rest.